5_fix_CountContribution.sol

```diff
@@ -22,6 +22,8 @@ contract CountContribution{
      *  @param _amount The amount of the contribution.
      */
     function recordContribution(address _user, uint _amount) {
+        require(contribution[_user] + _amount >= contribution[_user]);
+        require(totalContributions + _amount >= totalContributions);
```

Comment:
I don't sure the second check is necessary. If the first test is okay, the input amount is a valid number.